rosidl_runtime_c/include/rosidl_runtime_c/string_functions.h

```diff
@@ -55,6 +55,24 @@ ROSIDL_GENERATOR_C_PUBLIC
 void
 rosidl_runtime_c__String__fini(rosidl_runtime_c__String * str);
 
+/// Copy rosidl_runtime_c__String structure content.
+/**
+ * This functions performs a deep copy, as opposed to the shallow copy that
+ * plain assignment yields.
+ *
+ * \param[out] input a pointer to a rosidl_runtime_c__String structure
```

Comment:
```suggestion
 * \param[in] input a pointer to a rosidl_runtime_c__String structure
```

Reply:
Fixed in https://github.com/ros2/rosidl/pull/650/commits/ab79f66aac8c5e876481596bb9cb06eb6a1e6e3b.